ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -210,14 +210,16 @@ define('setup_manage_taxonomies', ['reactaddons', 'lodash', 'jquery'],
               <label htmlFor="managed_vocabulary_type">
                 <input id="managed_vocabulary_type" type="radio"
                   name="vocabulary_type" value="m"
+                  checked={this.state.vocabulary_type === 'm'}
                   onChange={this.updateVocabularyType} />
                     Managed
               </label>
             </div>
             <div className="radio">
-              <label htmlFor="managed_vocabulary_type">
+              <label htmlFor="free_vocabulary_type">
```

Comment:
We shouldn't need the for attribute if the item being pointed to is contained inside the label element